assets/js/src/form_editor/store/actions.jsx

```diff
@@ -28,3 +28,20 @@ export function saveFormDone(result) {
     result,
   };
 }
+
+export function addNotice(content, status, isDismissible = false, id = null) {
```

Comment:
The same comment goes here, why would you use a boolean argument instead of two different actions?